### ISSUE TITLE
Incorrect reference to vaem:GMD_QUDT-SCHEMA

### ISSUE THREAD

**Issue by cpauloh** [qudt/qudt-public-repo#319]
The graph in the **_QUDT SHACL Schema (NoOWL)- Version 2.1.4_** document is referencing the VAEM graph metadata instance in the qudt schema document.

```
<http://qudt.org/2.1/schema/shacl/qudt> vaem:hasGraphMetadata vaem:GMD_QUDT-SCHEMA
```

**Comment by steveraysteveray** [comment]
Good catch. This is because we use the TopQuadrant tools to generate the SHACL version of the schema, but of course it can't know that it should also update that link. We'll need to document these manual remediation steps each time we generate the SHACL file.
